Python/omevv/v1/scripts/getManagementHostComplianceData.py

```diff
@@ -0,0 +1,122 @@
+            get_host_compliance.sync_detailed(uuid=self.uuid, client=self.client)
+        return response.parsed
+
+    def populate_headers(self, column_count):
+        sheet1.write(column_count,0,"hostid")
+        sheet1.write(column_count,1,"host name")
+        sheet1.write(column_count,2,"model")
+        sheet1.write(column_count,3,"service tag")
+        sheet1.write(column_count,4,"console id")
+        sheet1.write(column_count,5,"console entity id")
+        sheet1.write(column_count,6,"hypervisor")
+        sheet1.write(column_count,7,"idrac ip")
+        sheet1.write(column_count,8,"idrac firmware version")
+        sheet1.write(column_count,9,"idrac license type")
+        sheet1.write(column_count,10,"idrac license description")
+        sheet1.write(column_count,11,"idrac license expiration date")
+        sheet1.write(column_count,12,"license entitlement id")
+        sheet1.write(column_count,13,"idrac license status")
+        sheet1.write(column_count,14,"snmp trap status")
+        sheet1.write(column_count,15,"state")
+        sheet1.write(column_count,16,"connection state")
+
+if __name__ == "__main__":
+    PARSER = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    PARSER.add_argument("--ip", "-i", required=True, help="OME Appliance IP")
+    PARSER.add_argument("--vcusername", "-u", required=True,
+                        help="username of vcenter")
+    PARSER.add_argument("--vcpassword", "-p", required=True,
+                        help="password of vcenter")
+    PARSER.add_argument("--vcUUID", "-d", required=True, default=None,
+                        help="UUID of the relevant vCenter")
+    PARSER.add_argument("--compliance_filter", "-g", required=True, default=None, help="compliance_filter")
+    ARGS = PARSER.parse_args()
+
+    if ARGS.ip is not None and ARGS.vcusername is not None and ARGS.vcpassword is not None and ARGS.vcUUID is not None and ARGS.compliance_filter is not None:
+        base_url = 'https://{ip}/omevv/GatewayService/v1/'.format(ip=ARGS.ip)
+        credential = Credential(username=ARGS.vcusername, password=ARGS.vcpassword)
+        output=HostManagementComplianceWrapper(base_url=base_url, vcUsercredential=credential, vCenterUUID=ARGS.vcUUID, compliance_filter=ARGS.compliance_filter).get_managed_hosts_compliance()
+        
+        book = xlwt.Workbook()
+        sheet1 = book.add_sheet('sheet1', cell_overwrite_ok=True)
+        
+        column_count = 0;
+        data_present = False;
+        HostManagementComplianceWrapper(base_url=base_url, vcUsercredential=credential, vCenterUUID=ARGS.vcUUID, compliance_filter=ARGS.compliance_filter).populate_headers(column_count)  
+        column_count = column_count + 1;
+        for row_num,x in enumerate(output):
+            if ARGS.compliance_filter == x.state:
+                data_present = True;
+                sheet1.write(column_count, 0, x.hostid)
```

Comment:
Can we have common way of writing to csv, refer https://github.com/dell/omivv/pull/37/files

Reply:
I think exporting to excel for this particular script would have better readability. I'll discuss with you.